addons/sale_product_configurator/static/src/js/product_configurator_modal.js

```diff
@@ -179,7 +179,7 @@ var OptionalProductsModal = Dialog.extend(ServicesMixin, VariantMixin, {
         var products = [this.rootProduct];
         this.$modal.find('.js_product.in_cart:not(.main_product)').each(function () {
             var $item = $(this);
-            var quantity = parseInt($item.find('input[name="add_qty"]').val(), 10);
+            var quantity = parseFloat($item.find('input[name="add_qty"]').val().replace(',', '.') || 1);
```

Comment:
It doesn't fail, but the qty is rounded to 1.0.
I tried removing the replace part, but parseFloat doesn't like american amounts either, the only change is that the resulting quantity is 1.25 instead of 1.0.

Note that TDE is not responsible for the scope and didn't do anything on this PR. Please avoid tagging people in messages, unless you do not have any answer the standard way (Codeowners, people managing a given PR, ...)